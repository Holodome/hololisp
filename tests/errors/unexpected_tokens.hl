(12312

  фывофыолв

  фыв)