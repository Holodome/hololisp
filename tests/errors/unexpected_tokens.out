cli:3:3: error: Unexpected token
  фывофыолв
  ^
cli:5:3: error: Unexpected token
  фыв)
  ^
2 errors generated.
